scripts/recreate.sh

```diff
@@ -1,7 +1,7 @@
 set -e -x
 
 bosh -n delete deployment webapp-warden
```

Comment:
In some rare cases it may cause problems if the release has changed. But I would suggest to keep it that way. We can either remove the -e option or add `| true`. This way the script won't exit even if the deployment is missing.